src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -83,7 +85,8 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
         }
       }
     } ~
-    path("admin" / "groups" / Segment / "members") { (groupName) =>
+    path("admin" / "groups" / Segment / "members") { (groupNameRaw) =>
+      val groupName = URLDecoder.decode(groupNameRaw, "UTF-8")
```

Comment:
for group names, do we limit them to UTF-8 when they are created?  We don't explicitly set UTF-8 anywhere else, just wondering if we might create any group names that are for instance UTF-16, and what happens if a path comes in here that is UTF-16